pxr/usd/ar/resolver.cpp

```diff
@@ -1444,7 +1444,7 @@ class _DispatchingResolver final
 
         bool HandlesFormat(const std::string& extension) const
         {
-            return _packageFormat == extension;
+            return _packageFormat == TfStringToLowerAscii(extension);
```

Comment:
For completeness we should also lower-case the extensions specified in the `plugInfo.json`, so that in the off chance that someone specifies a package resolver extension like `.UsDz` the case-insensitive comparison here still works.

The code that reads the extensions in is in `_InitializePackageResolvers` around line 1241. We should just make sure all of those extensions are also lower-cased.

Reply:
That looks great!